AndroidBound/src/main/java/solutions/alterego/androidbound/android/BoundFragmentDelegate.java

```diff
@@ -256,7 +256,7 @@ public void onDestroyView() {
                 && getViewBinder() != null
                 && boundActivityRef.getWindow() != null
                 && boundActivityRef.getWindow().getDecorView() != null) {
-            getViewBinder().clearBindingForViewAndChildren(getBoundActivity().getWindow().getDecorView().getRootView());
+            getViewBinder().clearBindingForViewAndChildren(getBoundActivity().getWindow().getDecorView());
```

Comment:
What if one is using BoundFragmentDelegate without using the BindableActivity? In this case the dispose(Context) won't be called. am I wrong ?